src/components/ChatList/ChannelListHeaderItem.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import FastImage from 'react-native-fast-image';
+import {StyleSheet, Text, View} from 'react-native';
+
+import dimen from '../../utils/dimen';
+import {colors} from '../../utils/colors';
+import {fonts, normalizeFontSize} from '../../utils/fonts';
+
+export interface ChannelListTabItemProps {
+  picture: string | null;
+  name: string;
+  type: 'SIGNED' | 'ANONYMOUS';
+  testID?: string;
+}
+
+const ChannelListHeaderItem = (props: ChannelListTabItemProps) => {
+  const {name, picture, type, testID} = props;
+
+  const styles = StyleSheet.create({
+    container: {
+      flexDirection: 'row',
+      height: dimen.normalizeDimen(48),
+      marginBottom: dimen.normalizeDimen(2),
+      paddingLeft: dimen.normalizeDimen(30),
+      paddingRight: dimen.normalizeDimen(30),
+      justifyContent: 'center',
+      alignItems: 'center',
+      backgroundColor: colors.white,
+      borderBottomWidth: 2,
+      borderBottomColor: type === 'SIGNED' ? colors.darkBlue : colors.anon_primary
```

Comment:
Should be call constant variabled SIGNED

Reply:
solved in new commit